Monika After Story/game/script-story-events.rpy

```diff
@@ -656,9 +657,9 @@ label mas_random_ask:
             m 1eua "Great!"
             m "If you get tired of watching me talk about the same things over and over,{w} just open up the settings and uncheck 'Repeat Topics'."
             # TODO: this really should be a smug or wink face
-            m "That tells me when {cps=*2}you're bored of me{/cps}{nw}"
+            m 1tua "That tells me when {cps=*2} you're bored of me.{/cps}{nw}"
             $ _history_list.pop()
-            m "That tells me when {fast}you just want to quietly spend time with me."
+            m 1eua "That tells me when {fast}you just want to quietly spend time with me."
```

Comment:
If you keep the extra space above, it's not going to line up with this next line.